package.json

```diff
@@ -34,14 +37,14 @@
         "build-all": "yarn build && yarn build-template && yarn build-website",
         "start": "yarn workspace babylonjs-editor start",
         "package": "yarn clean && yarn clean:node_modules && yarn && yarn lint && yarn build-all && yarn test && node build.mjs",
-        "clean": "rm -rf editor/build editor/declaration editor/electron-packages editor/coverage quixel/build quixel/declaration tools/build tools/declaration tools/coverage template/.next website/.next",
-        "clean:node_modules": "rm -rf ./node_modules ./editor/node_modules ./quixel/node_modules ./tools/node_modules ./template/node_modules ./website/node_modules",
+        "clean": "rimraf editor/build editor/declaration editor/electron-packages editor/coverage quixel/build quixel/declaration tools/build tools/declaration tools/coverage template/nextjs/.next template/solidjs/dist template/vanillajs/dist website/.next",
+        "clean:node_modules": "rimraf --glob **/node_modules",
```

Comment:
I saw that this command did not remove ALL node_modules. I had to run twice :(

Reply:
I checked it, and it remove all node_modules, just try to click "refresh explorer" button if you are using vscode after this command

Reply:
I'm trying again, maybe you are right ^^